### PR TITLE
create-react-play is unable to process style and props type for ts template

### PR DESCRIPTION
**Pull request by surya-situ**
Fixes: #33

### CODE REVIEW

File: lib/services/template/_templates/component_tsx.tmpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PlayHeader from 'common/playlists/PlayHeader';
-import './styles.{{style}}';
+import './styles.%PLAY_STYLE%';
 
 function %PLAY_TITLE_NAME_SANITIZED%(props) {
```
Comment (koustov) [inline lib/services/template/_templates/component_tsx.tmpl:5]: please add the type of `props`. I mean
```javascript
function %PLAY_TITLE_NAME_SANITIZED%(props:any) {
```

**Review by koustov — CHANGES_REQUESTED** [review]
A tiny review comment

File: lib/services/template/_templates/component_tsx.tmpl (hunk 1 above)
Comment (surya-situ) [reply lib/services/template/_templates/component_tsx.tmpl]: added props

**Review by koustov — APPROVED** [review]
lgtm

**Comment by koustov** [comment]
I have merged and published a new version (17) of the package

**Pull request merged**